functions/.gitignore

```diff
@@ -7,4 +7,5 @@ typings/
 
 # Node.js dependency directory
 node_modules/
-*.local
\ No newline at end of file
+*.local
+.env
```

Comment:
Did you intend to add this to the ignores? I can see that you added one to the root of the project and wondered if you wanted to follow the same pattern for functions.

Reply:
Good catch. I'll review this again. I can't remember offhand how envs work in the firebase functions but I know it's different than vite. I don't think it's hurting anything now and I'd like to have this code in main while dts is working on the deploy pipeline.